Sprint 2/UserValidation.java

```diff
@@ -0,0 +1,30 @@
+/*
+    Author: Logan Jackson
+    Description: UserValidation class is a class which contains
+    methods to validate user's inputs.
+*/
+

```

Comment:
Code is clean and logic sound. 
One suggestion is to review javadoc formatting.